Test/Web/FunctionalTests/TestApps/Aspx40/ApplicationInsights.config

```diff
@@ -44,8 +39,10 @@
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.UnobservedExceptionTelemetryModule, Microsoft.AI.WindowsServer"/>
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.FirstChanceExceptionStatisticsTelemetryModule, Microsoft.AI.WindowsServer"/>
   </TelemetryModules>
-  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel">
-    <EndpointAddress>http://localhost:4001/v2/track</EndpointAddress>
-    <MaxTelemetryBufferDelay>0:00:01</MaxTelemetryBufferDelay>
-  </TelemetryChannel>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel"/>
+  <TelemetryProcessors>
+    <Add Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.AdaptiveSamplingTelemetryProcessor, Microsoft.AI.ServerTelemetryChannel">
```

Comment:
This will break the Func tests. (Ai.Config is destroyed after every nuget upgrade) Please revert this file to original.